src/cljs/zetawar/data.cljs

```diff
@@ -133,6 +136,7 @@
                 :cost 75
                 :movement 9
                 :can-capture true
+                :can-repair false
```

Comment:
The armor classes (I guess I actually called them armor-types...) should be the same as those used for zoc and attack strength. Engineers should be able to repair 'armored' units and medics should be able to repair 'personnel'. All other units get an empty set.